docs/interfaces/cli-ug.rst

```diff
@@ -43,6 +43,8 @@ a `virtualenv <https://virtualenv.pypa.io/en/latest/>`__ and using the ``pip`` u
 
    pip install determined
 
+.. include:: ../_shared/pip-install-determined.txt
```

Comment:
https://docs.determined.ai/latest/interfaces/cli-ug.html#uninstalling

This is probably as good a time as any to fix the horribly-outdated uninstall. You can no longer uninstall the CLI with pip separate from the rest of determined (and I'm not quite sure why you'd want to).

Reply:
resolved

This command uninstalls the ``determined`` library, including the Determined CLI, from your system.